srcpkgs/cross-or1k-none-elf-gcc/template

```diff
@@ -0,0 +1,91 @@
+# Template file for 'cross-${_triplet}-gcc'
+_triplet=or1k-none-elf
+_pkgname=gcc
+pkgname=cross-${_triplet}-gcc
+version=10.2.0
+revision=1
+wrksrc="gcc-${version}"
+build_wrksrc=build
+build_style=gnu-configure
+make_build_args="INHIBIT_LIBC_CFLAGS='-DUSE_TM_CLONE_REGISTRY=0'"
+hostmakedepends="autoconf automake cross-or1k-none-elf-binutils bison flex
+ perl tar texinfo"
+makedepends="gmp-devel isl15-devel libmpc-devel mpfr-devel zlib-devel"
+depends="cross-or1k-none-elf-binutils"
+short_desc="GNU Compiler Collection"
+maintainer="John Sullivan <jsullivan@csumb.edu>"
+license="GFDL-1.2-or-later, GPL-3.0-or-later, LGPL-2.1-or-later"
+homepage="https://gcc.gnu.org"
+distfiles="${GNU_SITE}/gcc/gcc-${version}/gcc-${version}.tar.xz"
+checksum=b8dd4368bb9c7f0b98188317ee0254dd8cc99d1e3a18d0ff146c855fe16c1d8c
+alternatives="or1k-none-elf:/usr/bin/or1k-none-elf-cc:/usr/bin/or1k-none-elf-gcc"
+nocross=yes
+nopie=yes
+nostrip_files="libgcc.a libgcov.a"
+
+_32bit_failed="https://travis-ci.org/github/void-linux/void-packages/jobs/717355887"
+case $XBPS_TARGET_MACHINE in
+	i686*|armv*|mips*|ppc|ppc-musl) broken=${_32bit_failed} ;;
+esac
+
+post_extract() {
+	mkdir -p build
+}
+
+pre_configure() {
+	export CFLAGS_FOR_TARGET="-g -Os -ffunction-sections -fdata-sections"
+	export CXXFLAGS_FOR_TARGET="-g -Os -ffunction-sections -fdata-sections"
```

Comment:
While you're at it, move this in global scope, remove `pre_configure` and `pre_build`.

Reply:
Fixed